Sources/Testing/Testing.docc/ParameterizedTesting.md

```diff
@@ -1,4 +1,4 @@
-# Parameterized testing
+# Parameterizing testing
```

Comment:
"Parameterized testing" is a noun phrase here, sort of. It's a distinct concept (or subtopic) of "test" in general. I won't dwell on it, but it might be worth finding a way to preserve the term of art "parameterized testing" here. (Or maybe we just give up on it as a term of art entirely?)

Reply:
Maybe?

- "Running a test with multiple inputs"
- "Testing with different input values"

Something like that? Just drop the word "parameterized" so we don't confuse it for a verb later (it's an adverb in this position.)